src/openapi/tower.yml

```diff
@@ -61,7 +64,7 @@ model:
         description: UUID of user who launched job
         $ref: 'common.yml#/model/UUID'
       status:
-        description: Job status
+        description: Job statu
```

Comment:
Is this spelling intentional?  When I first saw it I assumed it was but then had to look it up to see that it is valid but not common.

Reply:
typo. accidentally pressed `x` I guess :D